packages/parse5/lib/tokenizer/index.ts

```diff
@@ -132,7 +132,6 @@ const enum State {
     AMBIGUOUS_AMPERSAND,
     NUMERIC_CHARACTER_REFERENCE,
     HEXADEMICAL_CHARACTER_REFERENCE_START,
-    DECIMAL_CHARACTER_REFERENCE_START,
```

Comment:
I believe that this should either be upstreamed to the HTML spec or that it should be documented very clearly (in each line of removal) that this is a deviation from the spec and why. Even if the outcome is the same, the spec is complex, and people looking for bugs will notice that a state is missing.

Reply:
I've opened https://github.com/whatwg/html/pull/7841 to upstream this change.

Reply:
sweet, thanks!